### ISSUE TITLE
Positron logo text unreadable in dark mode

### ISSUE THREAD

**Issue by jmcphers** [posit-dev/positron#8197]
#### Positron and OS details:

```
Positron Version: 2025.07.0 (Universal) build 134
Code - OSS Version: 1.100.0
Commit: 572a37b9b646af0b813dbe4d424bc9bfa1629646
Date: 2025-06-18T03:33:46.852Z
Electron: 34.5.1
Chromium: 132.0.6834.210
Node.js: 20.19.0
V8: 13.2.152.41-electron.0
OS: Darwin arm64 24.5.0
```

#### Interpreter details:

N/A

## Describe the issue:

Positron's new logo text is always black, with the result that it is practically unreadable in dark mode.

<img width="785" alt="Image" src="https://github.com/user-attachments/assets/dd974f31-94e7-4777-86c0-40f25c838ceb" />

## Expected behavior

Readable logo. Maybe we should switch this to light blue or white in dark mode?

**Comment by jthomasmock** [comment]
@paul-dicristina - I think we'll need an alternate SVG here.

**Comment by paul-dicristina** [comment]
Dark mode SVG: 

![Image](https://github.com/user-attachments/assets/9b1ec2b5-c14c-4621-920d-b4c6bfcd72ce)

**Comment by rodrigosf672** [comment]
### Verified Fixed
    Positron Version(s) : 2025.08.0 (Universal) build 1
    OS Version          : Sequoia 15.5

### Test scenario(s) 
- Logo is readable in Dark mode.

![Image](https://github.com/user-attachments/assets/0e974448-7bae-4bff-a160-1e4edf758ce2)
